libraries/standard/http/utest/http_utest.c

```diff
@@ -1162,8 +1172,8 @@ void test_Http_ReadHeader_Header_Not_In_Response( void )
     /* Configure the http_parser_execute mock. */
     invokeHeaderFieldCallback = 1u;
     invokeHeaderValueCallback = 1u;
-    pFieldLocToReturn = &pTestResponse[ headerFieldInRespLoc ];
-    fieldLenToReturn = strlen( HEADER_NOT_IN_BUFFER );
+    pFieldLocToReturn = &pTestResponse[ otherHeaderFieldInRespLoc ];
```

Comment:
(Minor) Please also add a check to ensure that the length of `otherHeaderFieldInRespLoc ` and `HEADER_NOT_IN_BUFFER` are the same (to guard against future changes)

Reply:
Nice suggestion